src/Components/StaticCollapse.tsx

```diff
@@ -24,6 +24,10 @@ export class StaticCollapse extends React.Component<{ open: boolean }> {
     }
   }
 
+  shouldComponentUpdate(nextProps) {
```

Comment:
fixes a bug where the collapse was animating a close animation when initially mounting because it was being rendered twice somehow.